pkg/config/linters_settings.go

```diff
@@ -184,6 +185,12 @@ type LintersSettings struct {
 	Custom map[string]CustomLinterSettings
 }
 
+type AsasalintSettings struct {
+	Exclude          []string `mapstructure:"exclude"`
+	NoDefaultExclude bool     `mapstructure:"no_default_exclude"`
+	IgnoreInTest     bool     `mapstructure:"ignore_in_test"`
```

Comment:
```suggestion
	NoDefaultExclude bool     `mapstructure:"no-default-exclude"`
	IgnoreInTest     bool     `mapstructure:"ignore-in-test"`
```

Reply:
thanks, I am juts reviewing the CI to find the bug 😂

Reply:
this is not related to the CI problem.

Reply:
the ci failed because `any` not in go 1.17 😢 

```
Error:         	            	asasalint.go:6:28: undeclared name: `any` (typecheck)
```